docs/source/index.rst

```diff
@@ -0,0 +1,282 @@
+**Zoomus 1.1.5**
+================
+
+|Build Status|
+
+Python wrapper around the Zoom.us REST API v1 and v2. This work is heavily inspired by the Ruby GEM of the same name, Zoomus
+
+.. |Build Status| raw:: html
+
+    <a href="https://travis-ci.org/github/prschmid/zoomus">
+        <img src="https://travis-ci.org/prschmid/zoomus.svg?branch=develop"/>
+    </a>
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+
+
+Contents
+========
+
+1. :ref:`Installation`.
+2. :ref:`Usage`.
+3. :ref:`Create user using API V2`.
+4. :ref:`Zoom User Methods provided by zoomus client`
+5. :ref:`Zoom Meeting Methods provided by zoomus client`.
+6. :ref:`Zoom Report Methods provided by zoomus client`.
+7. :ref:`Zoom Webinar Methods provided by zoomus client`.
+8. :ref:`Zoom Phone Methods provided by zoomus client`.
+9. :ref:`Zoom Groups Methods provided by zoomus client`.
+10. :ref:`Running the Tests`.
+11. :ref:`Contributing`.
+
+.. _Installation:
+
+**Installation**
+----------------
+
+To install zoomus::
+
+    $ pip install zoomus
+
+.. _Usage:
+
+**Usage**
+---------
+
+Initialize client with API_KEY and API_SECRET
+*********************************************
+
+As Zoom's default is now the V2 API, the client will default to the V2 version of the API
+
+*For API V2*
+############
+
+::
+
+  import json
+  from zoomus import ZoomClient
+
+  client = ZoomClient('API_KEY', 'API_SECRET')
+
+*For API V1*
+#############
+
+Zoom has yet to officially remove support for the V1 API, and so to use the V1 API one can instantiate a client as follows.
+
+::
+
+  import json
+  from zoomus import ZoomClient
+
+  client = ZoomClient('API_KEY', 'API_SECRET', version=1)
+
+
+*Using with a manage context*
+#############################
+
+::
+
+  with ZoomClient('API_KEY', 'API_SECRET') as client:
+    user_list_response = client.users.list()
+
+
+.. _Create user using API V2:
+
+Create user using API V2:
+*************************
+
+*Refer Create User* `https://marketplace.zoom.us/docs/api-reference/zoom-api/users/usercreate`
+
+::
+
+  params = {
+    "action": "create",
+    "user_info": {
+      "email": "dhjdfkghdskjf@fgkjfdlgjfkd.gh",
```

Comment:
Can you make this a more appealing email? E.g. "test@test.com"

Reply:
Update to example@test.com